docs/Search.md

```diff
@@ -18,6 +18,7 @@ Surround `value` in quotation marks to use whitespaces within it, useful for dea
     - `timeout` or `ban` - filters moderation messages
     - `highlighted` - filters highlighted messages
     - `system` - filters system messages (grey text ones like: "Now hosting username", "streamer is live", etc.)
+    - `first-msg` - filters first messages (added in [nightly release](../Help/#what-is-nightly-and-how-to-use-install-it))
```

Comment:
should say 2.3.5 instead of nightly, but I cant decide on a good structure to include both 2.3.5 and a minor description. so it can wait to be changed until it's reviewed.

```suggestion
    - `first-msg` - filters first messages [a users first message in the channel], (added in 2.3.5).
```
`[]()` vs `()()` or 2.3.5 mention first, or full description like `regex:` below🤷

Reply:
i was just copying the way it was done at line 9 of this file:
```Surround `value` in quotation marks to use whitespaces within it, useful for dealing with the regex filter (added in [nightly release](../Help/#what-is-nightly-and-how-to-use-install-it))```

i dont think it should say 2.3.5 since it wasnt added in 2.3.5, its still/will be in nightly

Reply:
also if you would like a better description, hows this?
```suggestion
    - `first-msg` - filters a user's first message in the channel (added in [nightly release](../Help/#what-is-nightly-and-how-to-use-install-it))
```

Reply:
Mistake on my part, despite the fact that I clicked this PR from the currently open PR, I immediately thought `first-msg = 2.3.5` since it's the most obvious feature released, but yeah nightly xd.